pkg/kubelet/container_manager_linux.go

```diff
@@ -125,3 +152,48 @@ func getContainer(pid int) (string, error) {
 
 	return cgroups.ParseCgroupFile("cpu", f)
 }
+
+// Ensures the system container is created and all non-kernel processes without
+// a container are moved to it.
+func (cm *containerManagerImpl) ensureSystemContainer() error {
+	// Move non-kernel PIDs to the system container.
+	for {
```

Comment:
We should add a max retry to keep kubelet from hanging forever if we somehow fail to move a pid.